desktop/libs/notebook/src/notebook/urls.py

```diff
@@ -27,7 +27,7 @@
 
 # Views
 urlpatterns = [
-  re_path(r'^$', notebook_views.notebook, name='index'),
+  re_path(r'^$', notebook_views.notebook, name='index',
```

Comment:
API Reviewers Assemble - Review Needed for the Hue API 
cc @tabraiz12
Refer API GuideLines -  https://cloudera.atlassian.net/wiki/spaces/ENG/overview?homepageId=101482622